compiler/control/OMROptions.cpp

```diff
@@ -1458,12 +1465,17 @@ OMR::Options::setString(char *option, void *base, TR::OptionTable *entry)
    return dummy_string;
    }
 
+char *
+OMR::Options::setStringInJitConfig(char *option, void *base, TR::OptionTable *entry)
```

Comment:
This is a new function.  What is the purpose of the `base` parameter because its unused here and in all implementations in this PR?  Please add a Doxygen-style comment (perhaps in the header file where the other comment is already).